index.css

```diff
@@ -0,0 +1,165 @@
+@import url(https://fonts.googleapis.com/css?family=Playfair+Display|Poppins);
+
+body {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.a-logo {
+  display: flex;
+  justify-content: end;
+  height: 100%;
+  width: 100%;
+}
+
+nav a {
+  display: none;
+}
+
+header > * {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.single-shape .triangle {
+  background-image: url(images/triangle.png);
+  height: 100%;
+  width: 100%;
+  background-repeat: no-repeat;
+}
+
+.single-shape .square {
+  margin-left: -40px;
+  background-image: url(images/square.png);
+  height: 100%;
+  width: 100%;
+  background-repeat: no-repeat;
+}
+
+.fa-bars {
+  width: 300px;
+  color: black;
+  height: 100%;
+  margin-left: 70%;
+  margin-top: -50px;
+  cursor: pointer;
+  outline: none;
+  font-size: 30px;
+}

```

Comment:
- [ ] Kindly, let us modify or increase our margin-left size to look like the same we have on the figma

![port6](https://user-images.githubusercontent.com/55635977/129165288-a0f29786-2d95-4f57-83a2-853dae9e734f.png)